presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeSimpleQueries.java

```diff
@@ -57,9 +70,26 @@ public void testAggregations()
     }
 
     @Test
-    public void testJoins()
+    public void testJoin()
+    {
+        assertQuery("SELECT * FROM orders o, lineitem l WHERE o.orderkey = l.orderkey AND o.orderkey % 2 = 1");
+    }
+
+    @Test
+    public void testShuffleStats()
```

Comment:
No point in running same queries twice. Let's fold this logic into the testJoin.